src/univariates.jl

```diff
@@ -381,7 +382,10 @@ logccdf(d::UnivariateDistribution, x::Real) = log(ccdf(d, x))
 """
     quantile(d::UnivariateDistribution, q::Real)
 
-Evaluate the inverse cumulative distribution function at `q`.
+Evaluate the generalized inverse cumulative distribution function at `q`.
+
+For a given `0 ≤ q ≤ 1`, `quantile(d, q)` is the smallest value `x` in the support of `d`
```

Comment:
While I think it good to include the accurate definition, I feel like it would be helpful to state the simplified version before stating the precise but also more complicated definition. For most people, in most situations, the simple but slightly inaccurate definitions is probably more helpful. For the same reason, maybe put "generalized" in parentheses as many people might not know what it means. Same comment applies to the other changes.

Reply:
Done!